doc/internal/release/how-to-release.md

```diff
@@ -102,7 +99,7 @@ On the release day, there are several things to do:
 - **Tag the Vitess release.**
   > - A guide on how to tag a version is available in the [How To Release Vitess](#how-to-release-vitess) section.
 - **Update the release notes on `main`.**
-  > - During the code freeze, we created a Pull Request against `main` to update the release notes. It must be merged.
+  > - One Pull Request against `main` must be created, it will contain the new release notes that we are adding in the Release Pull Request.
```

Comment:
This is confusing. Are there two pull requests? One that this step is asking you to create, and another one that you are referring to as "Release Pull Request"?

Reply:
Yes there are. There is a "Release Pull Request" which is merged on the release branch. And there is another one which is made against `main` to update the release notes.